web/test/spec/filterbar/numeric-range-directive.spec.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+describe('driver.filterbar: Numeric Range', function () {
```

Comment:
Breaking this controller out will be more of a headache than it is worth since I'm requiring filterCtl from parent as the shared controller. I can probably test this logic with some jquery selections instead